iree/compiler/Dialect/Modules/Strings/IR/Ops.td

```diff
@@ -51,17 +61,39 @@ def IREESTRINGS_I32ToStringOp : Op<IREESTRINGS_Dialect, "i32_to_string", [NoSide
   let arguments = (ins I32:$value);
 
   let results = (outs
-    IREESTRINGS_String:$result
+    STRINGS_String:$result
   );
 }
 
-def IREESTRINGS_PrintOp : Op<IREESTRINGS_Dialect, "print"> {
+def STRINGS_ToStringOp : Op<STRINGS_Dialect, "to_string", [NoSideEffect]> {
```

Comment:
shouldn't it be called ToStringTensor? I would expect ToString to return a single string.

Reply:
I switched to ToStringTensor throughout the dialect.